datadog-service/0.15.0/artifacthub-pkg.yml

```diff
@@ -19,9 +19,11 @@ annotations:
 install: '## Execute the following steps to install datadog-service:
 
   ```bash 
-  export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com"
-  helm install -n keptn datadog-service https://github.com/keptn-sandbox/datadog-service/releases/download/0.15.0/datadog-service-0.15.0.tar.gz --set datadogservice.ddApikey=${DD_API_KEY} --set datadogservice.ddAppKey=${DD_APP_KEY} --set datadogservice.ddSite=${DD_SITE}
-  ```
+  export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com"  
```

Comment:
It seems like we are repeating the following line quite often in the README file. Maybe we can add it as a prerequisite to prevent code duplication (not related to your current changes).

Reply:
Hmm, we are using it at many places in the README. I will give it a thought (create a separate PR maybe?). Thank you @TannerGabriel. That being said, that's unrelated to this PR like you said. Can I get a stamp on this one? :pray:

Reply:
Created an issue: https://github.com/keptn-sandbox/datadog-service/issues/41